payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/MandateTextSpec.kt

```diff
@@ -18,12 +18,12 @@ data class MandateTextSpec(
     @StringRes
     val stringResId: Int
 ) : FormItemSpec() {
```

Comment:
Can we reuse MandateTextSpec, rather than adding a new CashAppPayMandateTextSpec?

Reply:
`CashAppPayMandateTextSpec` is using `MandateTextSpec` under the hood.

Reply:
I don't understand why we need this class though, couldn't we just add MandateTextSpec with the specific arguments in LpmRepository?

Reply:
We need `CashAppPayMandateTextSpec` because it passes `merchantName` to the underlying `MandateSpec` twice (as there are two placeholders in the string). We can’t do that with the plain `MandateSpec`.